src/material/core/ripple/_ripple.scss

```diff
@@ -30,7 +30,7 @@
     pointer-events: none;
 
     transition: opacity, transform 0ms cubic-bezier(0, 0, 0.2, 1);
-    transform: scale(0);
+    transform: scale(0.6);
```

Comment:
I'm a little iffy about this one since I think that starting it at zero looks better.

Reply:
Yeah, I'd agree. IMO It's hard to tell from [the spec](https://material.io/design/interaction/states.html#pressed) whether it should start at `0.6` or not but it might be better to match fully with MDCs implementation? 

Personally I'd also prefer keeping `scale(0)` as this "feels" a little better (subjectively obviously).

Reply:
Let's see what others think. It's worth noting that these numbers aren't directly from the spec, but from the MDC source code.